pkgs/development/python-modules/textblob/default.nix

```
@@ -0,0 +1,35 @@
+{
+  lib,
+  fetchPypi,
+  buildPythonPackage,
+  setuptools,
+  flit-core,
+  nltk,
+}:
+
+buildPythonPackage rec {
+  pname = "textblob";
+  version = "0.18.0";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-61B7Yr8ig6cfVr7T4PxO7H04jvdrA2mc+ZQWZXKo2vM=";
+  };
+
+  build-system = [ flit-core ];
+
+  dependencies = [ nltk ];
+
+  doCheck = true;
+
+  pythonImportsCheck = [ "textblob" ];
+
+  meta = with lib; {
```

```suggestion
meta = {
```
https://github.com/NixOS/nixpkgs/issues/208242

---

I have made the changes, should I make new pr for it? or is there other way?

---

Yes, you need to make a new PR